test/Microsoft.ML.Functional.Tests/SchemaDefinitionTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+using Microsoft.ML.RunTests;
+using Microsoft.ML.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Functional.Tests
+{
+    public class PredictionEngineScenarios : BaseTestClass
+    {
+        private MLContext _ml;
+
+        public PredictionEngineScenarios(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected override void Initialize()
+        {
+            base.Initialize();
+
+            _ml = new MLContext(42);
+            _ml.AddStandardComponents();
```

Comment:
Do you need to call `AddStandardComponents`? That should only be necessary when you are doing things like using the MAML syntax. When you are strictly using the API, it shouldn't be necessary.

Reply:
I added this line back, because of issue #2996.

---
In reply to: [266474155](https://github.com/dotnet/machinelearning/pull/2995#discussion_r266474155) [](ancestors = 266474155)